### PR TITLE
fix translation-missing in "Pushed by" for English locale (issue #2071)

### PR DESCRIPTION
**Pull request by Akrabut**
Added the two missing lines that caused the `translation_missing` element to be created.
[Issue reference](https://github.com/rubygems/rubygems.org/issues/2071)

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -292,6 +292,8 @@ en:
     gem_members:
       authors_header: Authors
       owners_header: Owners
+      pushed_by: Pushed by
+      yanked_by: Yanked by
```
Comment (sonalkr132) [inline config/locales/en.yml]: Can you please remove ones added [here](https://github.com/rubygems/rubygems.org/blob/9df7769b29baf4c65159ef7ae7824a9c1aeee789/config/locales/en.yml#L264).
Also, you will need to add/remove these keys from all locale files to make tests pass.

**Comment by Akrabut** [comment]
@sonalkr132 I fixed it, though it seems like the tests that fail for Travis pass locally for me.

**Comment by sonalkr132** [comment]
> 0c2350e

Please don't add translations in the same PR. Send a separate PR for this, we need independent translation review for this.
> 240f704, 11930fd

Squash it to in a single commit and add a nicer commit message, ex: Fix push and yank by i18n keys scope.

> 13f1d29

Please don't merge/pull master on your branch. Rebase your branch on updated master.

**Comment by Akrabut** [comment]
> Please don't add translations in the same PR. Send a separate PR for this, we need independent translation review for this.

These were the ones from the removed keys in the Spanish locale so I copied them to the correct keys, should I revert it and leave the new keys empty?

File: config/locales/es.yml [hunk 2]
```diff
@@ -300,7 +298,9 @@ es:
       header: "dependencias de %{title}"
     gem_members:
       authors_header: Autores
-      owners_header: Propietarios
+      owners_header: Administradores
```
Comment (sonalkr132) [inline config/locales/es.yml]: Sorry, didn't notice that push and yank by translation were already there. Undo `owners_header` change only. push and yank by is fine.

**Comment by Akrabut** [comment]
moved [here](https://github.com/rubygems/rubygems.org/pull/2093/)

**Pull request closed without merging**